test/app-luatest/fiber_slice_test.lua

```diff
@@ -1,8 +1,13 @@
 local clock = require('clock')
 local fiber = require('fiber')
+local ffi = require('ffi')
 local server = require('luatest.server')
 local t = require('luatest')
 
+ffi.cdef[[
+bool small_malloc_impl;
+]]
```

Comment:
This definition doesn't affect the test executed on a remote instance.

Reply:
Removed.